src/EnergyPlus/WaterThermalTanks.cc

```diff
@@ -4918,124 +5001,125 @@ void WaterThermalTankData::setupOutputVars(EnergyPlusData &state)
     // this->setupZoneInternalGains();
 }
 
-void WaterThermalTankData::setupChilledWaterTankOutputVars(EnergyPlusData &state)
+void WaterThermalTankData::setupChilledWaterTankOutputVars(EnergyPlusData &state, std::string ChilledOrHotKw)
```

Comment:
std::string_view

Reply:
I'll change the name to setupWaterTankOutputVars as there's now hot water tank as well, and use string_view for the input arg